src/open-api.ts

```diff
@@ -7,6 +7,14 @@ import {
 } from "./open-api-helpers";
 import { Routing, routingCycle, RoutingCycleParams } from "./routing";
 
+const convertColonToBrackets = (path: string): string =>
+  path
+    .split('/')
+    .map(segment =>
+        segment[0] === ':' ? `{${segment.substr(1)}}` : segment,
```

Comment:
It's possible that path parameter is used not in the beginning of the segment.
Example: `/flights/:from-:to` from https://expressjs.com/en/guide/routing.html.

That's why I'm using a regular expression recommended on the same page in order to obtain the path params.
Consider using `String.replace(//)` with the same RegExp I use in the [common helper getRoutePathParams()](https://github.com/RobinTail/express-zod-api/blob/c3f77c77a54eb87513a377c7c23adeeebc9954b2/src/common-helpers.ts#L220).